promptulate/tools/file/tools.py

```diff
@@ -2,6 +2,7 @@
 import shutil
 
 from promptulate.tools.base import Tool
+from pydantic import Field
```

Comment:
use `from promptulate.pydantic_v1 import BaseModel, Field` to import pydantic. In order to compatible pydantic v1 and pydantic v2.

ref: https://github.com/Undertone0809/promptulate/pull/483